pkgs/top-level/release-cross.nix

```diff
@@ -17,7 +17,12 @@
 , # Strip most of attributes when evaluating to spare memory usage
   scrubJobs ? true
 , # Attributes passed to nixpkgs. Don't build packages marked as unfree.
-  nixpkgsArgs ? { config = { allowUnfree = false; inHydra = true; }; }
+  nixpkgsArgs ? {
+    config = {
+      allowUnfree = false;
```

Comment:
Is there any reason `allowAliases = false` wasn't added for this one?

Reply:
Oops, must have gotten distracted.